src/expiring_market.sol

```diff
@@ -14,7 +14,7 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-pragma solidity ^0.4.18;
+pragma solidity >=0.4.18;
```

Comment:
I would recommend locking this to a specific version. For DSS we used 0.5.12

Reply:
Maybe locking to a specific one is too much but at least we could say ^0.5.12.
Solc 0.4 is already too old.